packages/rest/package.json

```
@@ -35,7 +35,8 @@
     "routing-controllers": "^0.9.0",
     "routing-controllers-openapi": "^3.1.0",
     "swagger-ui-express": "^4.1.6",
-    "typedi": "^0.10.0"
+    "typedi": "^0.10.0",
+    "node-fetch": "2.2.0"
```

Er is al en hogere versie beschikbaar:

```suggestion
    "node-fetch": "^2.6.7"
```